pkgs/build-support/bintools-wrapper/default.nix

```diff
@@ -33,6 +33,26 @@
 , useMacosReexportHack ? false
 , wrapGas ? false
 
+, defaultHardeningFlags ? with stdenvNoCC; [
```

Comment:
Is there an easy way to make this extensible?

Reply:
Arguably because it's exposed as a passthru you can always do e.g. `prev.override { defaultHardeningFlags = prev.defaultHardeningFlags ++ ["foo"]; }`